src/main/java/com/tobiasdiez/easybind/EasyObservableList.java

```diff
@@ -68,6 +69,19 @@ default BooleanBinding isEmptyBinding() {
         filteredList.predicateProperty().bind(predicate);
         return filteredList;
     }
+    
+	/*
+	 * Sadly, we cannot easily let {@link FilteredList} implement {@link EasyObservableList}, because that class is final. Neither can we return
+	 * both at once, as intersection types do not exist in Java. Therefore, the easiest way is to simply create two methods, one for each return
+	 * type.
+	 */
+    default EasyObservableList<E> filteredWrapped(ObservableValue<? extends Predicate<E>> predicate) {
+    	return EasyBind.wrapList(filtered(predicate));
+    }
+
+    default <U> EasyObservableList<U> mapped(Function<? super E, ? extends U> f) {
```

Comment:
Please rename this method to `map`, add a bit of JavaDoc and reuse https://github.com/tobiasdiez/EasyBind/blob/45106f173f638a74320bf9a9df26d527a11f133a/src/main/java/com/tobiasdiez/easybind/EasyBind.java#L294 in the implementation.

Reply:
I opted for `mapped` instead of `map` to be consistent with `filtered` and `sorted`. But personally I don't have any preference.

Reply:
Good observation. For consistency, I would then rename `filtered` to `filter`. This is also in line with the naming in Streams. Ahh, please also add a change log entry. Thanks

Reply:
"I would then rename filtered to filter"

um, there is `ObservableList#filtered` so renaming `filtered` to `filter` would break the consistency here …

Reply:
On second thought, `mapped` etc make indeed more sense since they really create new lists (instead of sorting a stream). So leave it like that but please add the JavaDoc and reuse `EasyBind.map`. Thanks!